test/test_utils/ecr.py

```diff
@@ -109,6 +124,38 @@ def get_ecr_image_scan_severity_count(ecr_client, image_uri):
     return severity_counts
 
 
+def get_all_ecr_image_scan_results_using_pagination(ecr_client, image_uri, scan_info_finding_key="enhancedFindings"):
```

Comment:
If you use the pagination technique defined in the boto3 ECR client documentation page, the size of this function can be significantly reduced.

Also, specifying that this uses pagination in the function name seems redundant, because it is the only correct way of getting the image scan results, when writing a helper function for it.

Reply:
Done